code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -160,8 +160,59 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "pineapple" = 2, "ham" = 2)
 	foodtype = GRAIN | VEGETABLES | DAIRY | MEAT | FRUIT | PINEAPPLE
 
+/obj/item/reagent_containers/food/snacks/pizza/arnold
+	name = "arnold pizza"
+	desc = "Hello you've reached arnold's pizza shop. I'm not here now I'm out killing pepperoni."
+	icon_state = "arnoldpizza"
+	slice_path = /obj/item/reagent_containers/food/snacks/pizzaslice/arnold
+	bonus_reagents = list("nutriment" = 30, "vitamin" = 6, "iron" = 10, "omnizine" = 30)
+	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "pepperoni" = 2, "9 millimeter bullets" = 2)
+
+/obj/item/reagent_containers/food/snacks/proc/try_break_off(mob/living/M, mob/living/user) //maybe i give you a pizza maybe i break off your arm
+	var/obj/item/bodypart/l_arm = user.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/r_arm = user.get_bodypart(BODY_ZONE_R_ARM)
+	if(prob(50) && iscarbon(user) && M == user && (r_arm || l_arm))
+		user.visible_message("<span class='warning'>\The [src] breaks off [user]'s arm!!</span>", "<span class='warning'>\The [src] breaks off your arm!</span>")
+		if(l_arm)
+			l_arm.dismember()
+		else
+			r_arm.dismember()
+		playsound(user,pick('sound/misc/desceration-01.ogg','sound/misc/desceration-02.ogg','sound/misc/desceration-01.ogg') ,50, 1, -1)
```

Comment:
TRUE

Reply:
TRUE
but do it anyway

Reply:
`playsound(user,pick('sound/misc/desceration-01.ogg','sound/misc/desceration-02.ogg','sound/misc/desceration-01.ogg') ,50, TRUE, -1)`

4th param is boolean

Reply:
`playsound(user,pick('sound/misc/desceration-01.ogg','sound/misc/desceration-02.ogg','sound/misc/desceration-01.ogg') ,50, TRUE, -1)`

4th param is boolean

Reply:
ooooooooooooooh so that's the thing mama mia